ui/src/components/song/SongEditorDesktopView.tsx

```diff
@@ -292,6 +314,47 @@ const SongEditorDesktopView: FC<SongEditorProps> = ({ actionOnEditor }) => {
                   }}
                 />
               </Stack>
+
+              <FormControlLabel
+                sx={{ mt: 2 }}
+                label="Show Simplified Chords"
+                control={<Switch onChange={() => setShowSimplifiedChords(!showSimplifiedChords)} />}
```

Comment:
Dont think this toggle is necessary. maybe can add desc on what simplified chords are and what the purpose is

Reply:
hm thinking it can save space lol. u mean just show both of the text fields?